contracts/AcceleratingDistributor.sol

```diff
@@ -7,6 +7,30 @@ import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Multicall.sol";
 
+/// @notice The following interfaces and structs are used in a convenience function claimAndStake that allows
+//  new stakers to claim rewarded LP tokens from a MerkleDistributor and then stake them atomically. Structs
+//  are copied from the MerkleDistributor contracts which also contain further details on struct properties.
+
+struct Claim {
+    uint256 windowIndex;
+    uint256 amount;
+    uint256 accountIndex;
+    address account;
+    bytes32[] merkleProof;
+}
+struct Window {
+    bytes32 merkleRoot;
+    uint256 remainingAmount;
+    IERC20 rewardToken;
+    string ipfsHash;
+}
+
+interface IMerkleDistributor {
```

Comment:
Would it not be better to import this from the export where this contract is defined? Same goes for the structs

Reply:
Yup done